src/lib/mongoBackend/CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ SET (SOURCES
     TriggeredSubscription.cpp
     mongoConnectionPool.cpp
     mongoGetSubscriptions.cpp
-    mongoGetRegistrations.cpp
+    mongoRegistrationGet.cpp
```

Comment:
For "direct" operations (I mean, the ones not based in a mapping to existing NGSIv1 based operation) I think the naming criteria is "mongo" + verb + resource. Is this a causal change due to a matter of taste? Or it is strongly intentional?

Reply:
Actually, this is strongly intentional, as it follows the style guide (unfortunately it is just a Should ... https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/contribution_guidelines.md#s2-object-action-naming-convention).

The advantage with names like this is that all related files (mongoRegGet, mongoRegPost, mongoRegDelete, etc) are sorted together, instead of sorted by verb which is less interesting.
[ The file name is of course the same as the function name ].

Reply:
After reading S2 rule, it seems it refers to function and variable names... Does it also applied to file names? Maybe we should discuss this by skype.

Reply:
Moreover, if at the end the file name it changed you should also do a similar change to this one in the SET(HEADERS, ...) part of the file.

Reply:
Morever bis :), the original mongoGetRegistrations.h file is still there. Mabye you copied it to create mongoRegistrationGet.h but didn't removed the original one. Please have a look.

Reply:
The S2 rule, talks about names of functions, true. However, creating a file for **one** external function, which is how it should be done, the name of the file should of course be the name of the function. So, by extrapolation, the S2 rule actually talks about files names as well.

About the 'moreover 1', you mean to not use setXXX but xxxSet?
If so, yes, I agree, of course. I used those names (that I don't like), as other files use that kind of names.

And, about 'moreover 2', of course, that file should be removed. I didn't really realize those files were they. Can't say I needed them ...

Reply:
Fixed in 53980e464

Reply:
About "moreover 1", what I mean is that CMakeLists.txt  should be modified not only in the .cpp part, but also in the .h part.

Reply:
Yeah, I saw. Got compilation error about it :-)